catalog/transformer/catalog-transformer-preview/src/main/java/org/codice/ddf/transformer/preview/PreviewMetacardTransformer.java

```diff
@@ -40,7 +40,10 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
               .getAttribute(Extracted.EXTRACTED_TEXT)
               .getValue()
               .toString()
+              .replaceAll("<", "&lt;")
```

Comment:
Do we have a standard library/method to sanitize strings like this?

Reply:
Not that i'm aware of

Reply:
You can check `XMLUtils` but there's nothing there that supports string from string to string. Looks like a DOM needs to be built to leverage those utils. If this suits our needs let's stick with something simple. @garrettfreibott thoughts?

Reply:
❓ @alexaabrd Could we use this instead of doing it manually? 
https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html#escapeHtml4(java.lang.String)

Reply:
This is enough sanitizing for this specific scenario since we're just putting it into the body of the HTML, but it might be best to do a full HTML character encoding like @Lambeaux mentioned above just in case someone in the future uses this in a different context.

Reply:
Yeah I'd be more comfortable leaving it as it is now if it was possible to write a test that would fail if it was used out of context. But I think that's impossible.

Reply:
Updated to use `StringEscapeUtils.escpaceHtml4` from apache's commons-text  👍